tuna.go

```diff
@@ -745,6 +752,19 @@ func (c *Common) CreateServerConn(force bool) error {
 				return err
 			}
 
+			if c.minBalance > 0 {
+				entryToExitMaxPrice, exitToEntryMaxPrice, err := ParsePrice(c.ServiceInfo.MaxPrice)
+				if err != nil {
+					return err
+				}
+				if entryToExitMaxPrice > 0 || exitToEntryMaxPrice > 0 {
+					balance, err := c.Client.BalanceByAddress(c.Wallet.Address())
+					if err == nil && balance.ToFixed64() < c.minBalance {
```

Comment:
I would suggest we log the error if not nil

Reply:
Yes, modified.